### PR TITLE
diode_constant_for_plane_parallel_diode

### PR DESCRIPTION
**Pull request by Zulusssss**
add law and tests

### CODE REVIEW

File: symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The volt-ampere characteristic of a vacuum diode is described by the law of degree three of the second. The diode
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py:12]: Use common terminology. It's `current-voltage` characteristic.
And this law is better known as `3/2-power law`

File: symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## constant that is in this law depends only on the relative position, shape and size of the electrodes of the vacuum diode.
+## For a plane-parallel diode, the diode constant depends only on the area of the electrodes and the distance between them.
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py:14]: There is no valuable information in this description. One can look into the law, to know what it depends on.

File: symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py (hunk 1 above)
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py:12]: Vacuum diode is mentioned. Is it the same as plane parallel diode?

File: symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Law is: g = (4 / 9) * e0 * sqrt(2 * e / m) * (s / d^2), where
+## g - diode constant,
+## e0 - electric constant,
+## e - elementary charge,
+## m -  electron rest mass,
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py:20]:
```suggestion
## m - electron rest mass,
```

File: symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+## s - electrode area,
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py:21]: Area of one electrode or both?

File: symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py (hunk 1 above)
Comment (Zulusssss) [reply symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py:12]: > Vacuum diode is mentioned. Is it the same as plane parallel diode?

The vacuum diode may have different electrode geometries. Here we are talking about a vacuum diode with plane-parallel electrodes.

Comment (blackyblack) [reply symplyphysics/laws/electricity/circuits/diode_constant_for_plane_parallel_diode.py:12]: That's what should be in description.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**